src/main/kotlin/lotto/model/PurchasedTickets.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.model
+
+const val TICKET_PRICE = 1000;
+
+class PurchasedTickets(private val tickets: List<TicketModel>) {
+    fun getTickets(): List<TicketModel> = tickets
```

Comment:
In Kotlin, the getter is already included in the property! So we don't need to declare a function that just returns property value 🙂 

```suggestion
class PurchasedTickets(val tickets: List<TicketModel>) {
```

https://kotlinlang.org/docs/properties.html